pkg/kubelet/kuberuntime/labels.go

```diff
@@ -0,0 +1,264 @@
+/*
```

Comment:
Reviewed this file.
The original label code is quite unclear because of backward compatibility handling.
* We may still need to version and group pod/container annotation https://github.com/kubernetes/kubernetes/pull/25326.
* If we can make sure that people will drain node before upgrade, backward compatibility handling will be much easier.
Anyway, those are out of the scope of this PR.